CMakeLists.txt

```diff
@@ -1,13 +1,16 @@
-cmake_minimum_required (VERSION 3.1)
+cmake_minimum_required (VERSION 3.20)
 project(js50-profile-view)
 
-cmake_minimum_required (VERSION 3.20)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
 if(NOT DEFINED PINCHOT_API_ROOT_DIR)
   message(FATAL_ERROR 
   "PINCHOT_API_ROOT_DIR not defined!\n"
   "please set using -PINCHOT_API_ROOT_DIR=")
+else()
+  if(PINCHOT_API_ROOT_DIR STREQUAL "")
+	  message(FATAL_ERROR "PINCHOT_API_ROOT_DIR is empty, please provide a path to the pinchot API root directory")
```

Comment:
Small change recommended here; just to reduce wordiness drop "a" and capitalize "Pinchot":

>PINCHOT_API_ROOT_DIR is empty, please provide path to the Pinchot API root directory

Reply:
Also looks like tab indents aren't consistent. The styles are a bit different for C & .NET code, but the generally rule would be to be consistent with what is done in the file already.  Most of the C stuff is each indent is two spaces; no tab.